### ISSUE TITLE
Try out BLIS matrix operations

### ISSUE THREAD

**Issue by oysteijo** [oysteijo/simd_neuralnet#31]
It looks like [BLIS](https://github.com/flame/blis) is an actively developed matrix package written in C and optimized to several common architectures. I think maybe that can be just as effective as any BLAS implementation (OpenBLAS, GotoBLAS, Atlas or whatever there is).

This is just an idea, and maybe even a new project. I'm just typing down this as an idea.

**Comment by oysteijo** [comment]
Oh! Trying out examples of BLIS today, and it really doesn't look like something that is production ready yet.
I installed the blis-git packet from AUR, it built fine and I didn't get any errors, however the package didn't install `libblis.a` for my system for some reason. It just installed the dynamic libraries.

I really had to fight just to get the examples compiling. The makefile didn't get the suffix rules right, so I had to modify the Makefile. Then I had to copy the libblis.a file into the right directory. Then I was able to run the examples (tapi). Then I notices that running then raises a double-free exception.

Not very assuring, but the library looks promising. Maybe I can make a PR to these developers?
